.github/workflows/keep-alive.yaml

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Do we really need a scheduled workflow (and thus this keep-alive?).

Reply:
if we don't, the workflow is going to get disabled. And if the snap build breaks (change in snapcraft), we won't be able to know it.

Reply:
removed the keep alive flow and froze the snapcraft version